lib/svg/Info.tsx

```diff
@@ -8,6 +8,9 @@ export const InfoSVG = () => (
     width="100%"
     height="100%"
   >
-    <path d="M32 15c-2.5 0-4.5 2-4.5 4.5s2 4.5 4.5 4.5 4.5-2 4.5-4.5-2-4.5-4.5-4.5zm0 14.5c-2.5 0-4.5 2-4.5 4.5v12c0 2.5 2 4.5 4.5 4.5s4.5-2 4.5-4.5V34c0-2.5-2-4.5-4.5-4.5z" />
+    <path
+      className="primary"
```

Comment:
this `className` has been added, why?

Reply:
it's here for the testing, and it seems working okay with this attribute, so i decided to keep it for the other testing in the future too

Reply:
actually i will just remove it and add it in the future if it's needed for the simplicity